Fix input order in TRANSIT resampling tool and bump version to 3.2.3

dianichj opened the pull request:
Fixes the input order of control and experimental files in the `transit_resampling` tool.  
Also bumps the version to `3.2.3`.

- [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
- [x] License permits unrestricted use (educational + commercial)
- [ ] This PR adds a new tool or tool collection
- [x] This PR updates an existing tool or tool collection
- [ ] This PR does something else (explain below)

bgruening commented on tools/transit/transit_resampling.xml:
```diff
@@ -12,7 +12,7 @@
             ln -s '$filename' control_file_${idx}.wig &&
         #end for
-        transit resampling $input_files $control_files annotation.dat transit_out.txt
+        transit resampling $control_files $input_files annotation.dat transit_out.txt
```
can you please add a comment with reference to the upstream doc that the order is this way ...

dianichj replied:
`> python3 transit.py resampling <comma-separated .wig control files> <comma-separated .wig experimental files> <annotation .prot_table or GFF3> <output file> [Optional Arguments]`

In the upstream doc of the tool is already written that the order is first control and then experimental (input).

dianichj replied:
is this okay then? =) @bgruening

bgruening commented on tools/transit/transit_resampling.xml:
can you please add a profile version?

bgruening commented on tools/transit/transit_resampling.xml:
```diff
@@ -35,7 +35,8 @@
     <tests>
         <test>
-            <param name="inputs" ftype="wig" value="transit-in1-rep1.wig,transit-in1-rep2.wig" />
+            <param name="mode" value="Replicates" />
+            <param name="mode|inputs" ftype="wig" value="transit-in1-rep1.wig,transit-in1-rep2.wig" />
```
```suggestion
            <conditional name="mode">
                <param name="mode" value="Replicates" />
                <param name="inputs" ftype="wig" value="transit-in1-rep1.wig,transit-in1-rep2.wig" />
           </conditional>
```

something like that is needed

dianichj commented:
Hi @bgruening. 

Thanks a lot @RZ9082 for taking the time to help and teach me how to handle this bug! 

We did the following:
- remove the -r argument from the macros as resampling does not have it in the original documentation. 
- added the -r argument directly to the transit wrappers that do have it. 
- removed the handle replicates parameter from the resampling tool.
- updated macros: version and token.
- updated the output test file for the resampling tool.

bgruening commented:
Thanks! The tool will be updated over the next weekend automatically
